packages/grid/x-grid/src/XGrid.test.tsx

```diff
@@ -68,6 +68,7 @@ describe('<XGrid />', () => {
   });
 
   describe('keyboard', () => {
+    /* eslint-disable material-ui/disallow-active-element-as-key-event-target */
```

Comment:
Can we remove this? Is it difficult to fix the error?

Reply:
It's by design, there are rare cases, like this one, where the tradeoff is ok. See https://github.com/mui-org/material-ui/pull/23488.